apps/zbugs/shared/queries.ts

```diff
@@ -210,25 +215,28 @@ function buildListQuery(
     q = q.start(start);
   }
 
-  return q.orderBy(sortField, orderByDir).where(({and, cmp, exists, or}) =>
-    and(
-      open != null ? cmp('open', open) : undefined,
-      creator ? exists('creator', q => q.where('login', creator)) : undefined,
-      assignee
-        ? exists('assignee', q => q.where('login', assignee))
-        : undefined,
-      textFilter
-        ? or(
-            cmp('title', 'ILIKE', `%${escapeLike(textFilter)}%`),
-            cmp('description', 'ILIKE', `%${escapeLike(textFilter)}%`),
-            exists('comments', q =>
-              q.where('body', 'ILIKE', `%${escapeLike(textFilter)}%`),
-            ),
-          )
-        : undefined,
-      ...(labels ?? []).map(label =>
-        exists('labels', q => q.where('name', label)),
+  return q
+    .orderBy(sortField, orderByDir)
+    .orderBy('id', orderByDir)
```

Comment:
Bad diff from github here.  The only difference is the addition of  `.orderBy('id', orderByDir)`.

Embarrassingly this is what took the longest with this change.  Without this, we were almost but not exactly reversing the list with `orderByDir`, leading to lots of subtle bugs in the scrolling code when changing direction of paging.